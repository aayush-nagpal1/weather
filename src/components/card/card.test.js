import React from 'react';
import Card from './card';
import ReactDOM from "react-dom";
import renderer from 'react-test-renderer';

describe("Cards and SmallCards Component  ", ()=>{
  
  it('renders Cards and Small Cards component correctly', () => {
    const tree = renderer
      .create(<Card />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
})

