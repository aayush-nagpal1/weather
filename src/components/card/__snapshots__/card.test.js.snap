// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards and SmallCards Component   renders Cards and Small Cards component correctly 1`] = `null`;

exports[`Component 2 renders Cards component correctly 1`] = `null`;

exports[`test 1 renders correctly 1`] = `null`;
